enterprise/backend/test/metabase_enterprise/query_reference_validation/api_test.clj

```diff
@@ -53,18 +53,28 @@
                             ;; QFs not to include:
                             ;; - Field is still active
                             :model/QueryField {}             {:card_id     card-1
-                                                              :analysis_id qa-3
+                                                              :analysis_id qa-1
```

Comment:
Not sure why these were set inconsistently before. The card and analysis numbering should always match in these tests.